phoenix-core/src/main/java/org/apache/phoenix/schema/MetaDataClient.java

```diff
@@ -2051,7 +2087,7 @@ public MutationState addColumn(AddColumnStatement statement) throws SQLException
                     }
                 }
 
-                boolean isAddingPKColumn = false;
+                int numPkColumnsAdded = 0;
```

Comment:
I need numPkColumnsAdded to initialize the pkColumns list later . 
The boolean isAddingPKColumn can be derived from numPkColumnsAdded, so I removed it, I can add it back using the boolean makes it more readable

// create PK column list that includes the newly created columns
                    	List<PColumn> pkColumns = Lists.newArrayListWithExpectedSize(table.getPKColumns().size()+numPkColumnsAdded);